.circleci/config.yml

```diff
@@ -82,22 +73,35 @@ jobs:
             echo $GCLOUD_SERVICE_KEY > /tmp/gcloud.json
             export GOOGLE_APPLICATION_CREDENTIALS="/tmp/gcloud.json"
             # run the tests
-            run-go-tests --path test --timeout 60m | tee /tmp/logs/all.log
-          no_output_timeout: 3600s
+            mkdir -p /tmp/logs
+            run-go-tests --path test --timeout 2h | tee /tmp/logs/all.log
+          no_output_timeout: 1h
+
       - run:
+          name: parse test output
           command: terratest_log_parser --testlog /tmp/logs/all.log --outputdir /tmp/logs
           when: always
+
       - store_artifacts:
           path: /tmp/logs
       - store_test_results:
           path: /tmp/logs
 
 workflows:
   version: 2
-  build-and-test:
+  test:
     jobs:
-      - build
+      # We have to explicitly tell CircleCI to run on all tags and branches, or tag commits/pushes will not trigger
+      # builds: https://circleci.com/docs/2.0/workflows/#git-tag-job-execution.
+      - precommit:
+          filters:
+            branches:
+              only: /.*/
+            tags:
+              only: /.*/
```

Comment:
Doesn't this need the `context: - Gruntwork Admin`?

Reply:
Hi @rhoboat yes good catch! We obviously missed this repo and probably several other GCP ones on the auto-run. I'll make the updates manually now.